### PR TITLE
Fix line spacing for Swift error message.

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

**Comment by segiddins** [comment]
Should this have a test?

**Comment by dnkoutso** [comment]
Discussed offline, it does have one but uses `include` instead.

**Comment by dnkoutso** [comment]
will merge on green

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,21 +12,23 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
...
+  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
+  [#8024](https://github.com/CocoaPods/CocoaPods/pull/8024)
 
 
 ## 1.6.0.beta.1 (2018-08-16)
 
 ##### Enhancements
 
 * Every test spec will have its own xctest bundle.  
-  [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
```
[the remark this answers is not in the document]
Comment (dnkoutso) [reply CHANGELOG.md]: yea just cleared it up

File: lib/cocoapods/installer/xcode/target_validator.rb [hunk 2]
```diff
@@ -112,7 +112,7 @@ def verify_swift_pods_swift_version
           end.compact
 
           unless error_messages.empty?
-            raise Informative, "Unable to determine Swift version for the following pods:\n\n #{error_messages.join('\n')}"
+            raise Informative, "Unable to determine Swift version for the following pods:\n\n#{error_messages.join("\n")}"
```
[the remark this answers is not in the document]
Comment (dnkoutso) [reply lib/cocoapods/installer/xcode/target_validator.rb]: yeap it does!

**Pull request merged**